spyder/widgets/pathmanager.py

```diff
@@ -202,20 +215,58 @@ def setup(self):
         self.refresh()
 
     @Slot()
-    def synchronize(self):
+    def import_pythonpath(self):
+        """Import from PYTHONPATH environment variable"""
+        env_pypath = os.environ.get('PYTHONPATH', '')
+
+        if env_pypath:
+            env_pypath = env_pypath.split(os.pathsep)
+            env_pypath_msg = '<br>'.join(env_pypath)
+            qbox = QMessageBox(
+                QMessageBox.Question,
+                _("Import"),
+                _("Do you want to import the contents of your "
+                  "<tt>PYTHONPATH</tt> environment variable into Spyder?"),
+                QMessageBox.No | QMessageBox.Yes,
+                parent=self)
+            qbox.setInformativeText(_("The following paths will be imported:"
+                                      "<br><br>" + env_pypath_msg))
+            qbox.setDefaultButton(QMessageBox.Yes)
+            answer = qbox.exec()
+
+            if answer == QMessageBox.Yes:
+                spy_pypath = self.get_path_dict()
+                n = len(spy_pypath)
+
+                for path in reversed(env_pypath):
+                    if (path in spy_pypath) or not self.check_path(path):
+                        continue
+                    item = self._create_item(path)
+                    self.listwidget.insertItem(n, item)
+
+                self.refresh()
+        else:
+            QMessageBox.information(
+                self,
+                _("PYTHONPATH"),
+                _("<tt>PYTHONPATH</tt> is empty; nothing to import."),

```

Comment:
```suggestion
                _("Your <tt>PYTHONPATH</tt> environment variable is empty, so "
                  "there is nothing to import."),
```

Sorry to be so picky about these messages, but I think the more explicit we are in them, the easier would be for users to understand what this is about in case they don't know it.

For example, by saying that `PYTHONPATH` is an env var, it'd be easier for them to google for it.

Reply:
No problem.